vault/resource_kubernetes_auth_backend_role_test.go

```diff
@@ -433,6 +463,8 @@ func TestAccKubernetesAuthBackendRole_fullDeprecated(t *testing.T) {
 						"max_ttl", strconv.Itoa(newMaxTTL)),
 					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_role.role",
 						"period", "900"),
+					resource.TestCheckResourceAttr("vault_kubernetes_auth_backend_role.role",
+						"audience", newAudience),
```

Comment:
Same here for adding a test for what happens when "audience" isn't included.

Reply:
I think there are now tests already that tests basic config without audience.